docs/usage/beacon-management.md

```diff
@@ -33,7 +33,12 @@ You must generate a secret 32-byte (64 characters) hexadecimal string that will
 
 ### Configure Lodestar to locate the JWT secret
 
-When starting up a Lodestar beacon node in any configuration, ensure you add the `--jwt-secret $JWT_SECRET_PATH` flag to point to the saved secret key file.
+When starting up a Lodestar beacon node in any configuration, ensure you add the `--jwtSecret $JWT_SECRET_PATH` flag to point to the saved secret key file.
```

Comment:
Why change this flag? I think yargs supports both notations and the `--jwt-secret` flag is kind of standarized across clients

Reply:
> kind of standarized across clients

Kinda, but not really
- Teku: `--ee-jwt-secret-file` ([ref](https://github.com/dappnode/DAppNodePackage-teku/blob/master/beacon-chain/entrypoint.sh#L37))
- Lighthouse: `--execution-jwt` ([ref](https://github.com/dappnode/DAppNodePackage-lighthouse/blob/master/beacon-chain/entrypoint.sh#L51))
- Prsym: `--jwt-secret` ([ref](https://github.com/dappnode/DAppNodePackage-prysm/blob/main/beacon-chain/entrypoint.sh#L64))
- Nimbus: `--jwt-secret` ([ref](https://github.com/dappnode/DAppNodePackage-nimbus/blob/main/build/entrypoint.sh#L70))

For Lodestar, `--jwt-secret` is currently the only commonly used flag which uses different casing. I'd recommend we use the same standard (camelCase) for this flag as well.